pkg/networkmanager/interfaces.js

```diff
@@ -882,11 +882,13 @@ export function NetworkManagerModel() {
             activate: function (dev, specific_object) {
                 return call_object_method(get_object("/org/freedesktop/NetworkManager", type_Manager),
                                           "org.freedesktop.NetworkManager", "ActivateConnection",
-                                          objpath(this), objpath(dev), objpath(specific_object));
+                                          objpath(this), objpath(dev), objpath(specific_object))
+                        .then(([active_connection]) => active_connection);
```

Comment:
So I suppose this return value is not actually being used anywhere? At least right now? Otherwise it would be completely broken right now.

I.e. this PR is just to give us correct return values in case we ever start using them?